dotnet/src/SemanticKernel.Abstractions/KernelExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Orchestration;
+
+namespace Microsoft.SemanticKernel;
+
+public static class KernelExtensions
+{
+    public static Task<SKContext> RunAsync(
+        this IKernel kernel,
+        string skillName,
+        string functionName,
+        string? input = null,
```

Comment:
Should we remove this input all the way? 
Or just make a call to the args, passing the input as another argument in the dictionary.

Reply:
it's a good call, and one we'll need to decide soon. I've seen, heard, and made compelling arguments for both directions. I'd like to write down the pros and cons of each approach and get the architects together to hash it out. For now, I'm just trying to make it more clear what's what, so there's not a reliance on the "magic string" entry in the dictionary -- a secret special "INPUT" key.

Reply:
Reverted this file and calling format (reduced scope of PR).